Doc/library/curses.rst

```diff
@@ -1309,8 +1309,8 @@ the following methods and attributes:
    :meth:`refresh`.
 
 
-.. method:: window.vline(ch, n)
-            window.vline(y, x, ch, n)
+.. method:: window.vline(ch, n[, attr])
+            window.vline(y, x, ch, n[, attr])
 
    Display a vertical line starting at ``(y, x)`` with length *n* consisting of the
    character *ch*.
```

Comment:
```suggestion
   character *ch* with attributes *attr*.
```

Following wording from https://github.com/python/cpython/blame/894f2c3c161933bd820ad322b3b678d89bc2377c/Doc/library/curses.rst#L710 (also tested and *attr* does exist, and its values are per the table right under [this section](https://docs.python.org/3/library/curses.html#curses.ncurses_version).